test/test-signatures.ts

```diff
@@ -27,21 +31,23 @@ describe("Signatures", function () {
         //== Run the test in `func`
         samlObj.validatePostResponse(
           samlResponseBody,
-          tryCatchTest(done, function (error) {
+          tryCatchTest(done, function (error: any) {
             //== Assert error. If the error is `SAML assertion expired` we made it past the certificate validation
-            shouldErrorWith
-              ? error.should.eql(new Error(shouldErrorWith))
-              : error.should.eql(new Error("SAML assertion expired"));
+            error.should.eql(new Error(shouldErrorWith || "SAML assertion expired"));
```

Comment:
The way that this has been refactored, while it still works (in a non-obvious way), now seems to make the comment less useful. It seems that we should know when to expect which item in the test instead of saying both options are expected here.